libraries/botbuilder-dialogs-adaptive/src/actions/beginDialog.ts

```diff
@@ -23,7 +23,7 @@ export class BeginDialog<O extends object = {}> extends BaseInvokeDialog<O> {
     public constructor(dialogIdToCall: string, options?: O)
     
     /**
-     * Creates a new `BeginDialog` instance.
+     * Creates a new [BeginDialog](xref:botbuilder-dialogs-adaptive.BeginDialog) instance.
      * @param dialogIdToCall Optional. ID of the dialog to call.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.